pages/dkp/kaptain/2.1.0/tutorials/ngc/index.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: layout.pug
+navigationTitle: NGC catalog
+title: NVIDIA NGC catalog
+menuWeight: 30
+excerpt: Use NVIDIA's NGC catalog with Kaptain
+beta: false
+enterprise: false
+---
+
+Kaptain supports the usage of Nvidia's NGC catalog in networked environments, which you can access and set up via the Kubeflow UI. The NGC catalog includes containers, charts, pre-trained models, toolkits and more, which are optimized for GPU deployment and can now be used to run AI workflows in Kaptain.
+
+Since the NGC catalog is a constantly growing third-party catalog developed by NVIDIA, not all available images have been tested. Note that for some examples, you need to install additional tools like TAO Toolkit, DeepStream SDK, NVIDIA TensorRT and others.
+
+You can find all the available resources for your AI/ML workflow in the [NVIDIA NGC catalog][NGC_catalog], and detailed instructions on how to use these resources in the [NGC documentation][NGC_docs] site.
```

Comment:
I wonder about the word 'all' here, my first reading suggested that it meant I could find everything I would ever need in this catalog, which I doubt is true.    :-) 

```suggestion
You can find resources for your AI/ML workflow in the [NVIDIA NGC catalog][NGC_catalog], and detailed instructions on how to use these resources in the [NGC documentation][NGC_docs] site.
```